HIP/hip-513.md

```diff
@@ -0,0 +1,293 @@
+---
+hip: 513
+title: Smart Contract Traceability Extension
+author: Stoyan Panayotov <stoyan.panayotov@limechain.tech>, Mustafa Uzun <mustafa.uzun@limechain.tech>
+type: Standards Track
+category: Service
+needs-council-approval: Yes
+status: Draft
+last-call-date-time:
+created: 2022-06-08
+discussions-to:
+updated:
+requires: 206, 435, 260
+---
+
+## Abstract
+
+This HIP overrides [HIP-260](./hip-260.md) by moving traceability info from `ContractFunctionResult` to sidecar.
```

Comment:
There's an assumption of sidecar here. The term is an overloaded one and also ambiguous depending on experience.
Is there a link that can provided or a quick sentence to give context on what that is for community readers.

Reply:
You can reference [HIP-435](./hip-435.md) here